datasrc/languages/turkish.txt

```diff
@@ -2053,40 +2053,40 @@ Everything (%d)
 
 {durak-name}
 Waiting…
-== 
+== Bekleniyor…
 
 {durak-name}
 Players - %d/%d
-== 
+== Oyuncular - %d/%
 
 {durak-name}
 Select attack
-== 
+== Atak seç
 
 {durak-name}
 Attack!
-== 
+== Atak!
 
 {durak-name}
 Defend
-== 
+== Savun
 
 {durak-name}
 Pass!
-== 
+== Geç!
 
 {durak-name}
 End move
-== 
+== Hamleyi bitir
 
 {durak-name}
 Take cards
-== 
+== Kartları al
 
 {durak-name}
 Passed…
-== 
+== Geçildi…
 
 {durak-name}
 Next move: %ds
-== 
+== Sonraki hamle: %ds
```

Comment:
This is too long.
Player name is 15 characters max. (this would be 17)
In german, I used `Continue in: `, because its shorter in german.
`%d` equals one character, because its a number placeholder (countedown, 1-5).